### ISSUE TITLE
Labes problem

### ISSUE THREAD

**Issue by DuyunliangToon** [hnuzhy/SSDA-YOLO#19]
Hi, I have a question about tags while reproducing your paper. According to the description in your dataset configuration file, train_source_real, train_source_fake and test_traget_real require labels, and train_target_real and train_target_fake do not require labels. But in my actual training, I found that train_source_fake does not need labels, and train_target_real needs labels. From this point of view, it requires all the labels of the two data sets, which is not the so-called semi-supervised training. Not sure if I made a mistake, hope you have time to help me out.
![image](https://github.com/hnuzhy/SSDA-YOLO/assets/109786505/7819cc66-c9f0-4b61-ba64-e5047a7e8f75)

**Comment by hnuzhy** [comment]
Yes, we do use labels of images in `train_source_real` and `train_source_fake`, and not use labels of images in `train_target_real` and `train_target_fake`. When applying dataloading, the `train_source_fake` has exactly the same labels as `train_source_real`, thus we do not need load its labels one more time. As for labels of `train_target_real` and `train_target_fake`, we do not  use them when training, You can refer the training code in lines list below
https://github.com/hnuzhy/SSDA-YOLO/blob/master/ssda_yolov5_train.py#L489~#L534
We want these labels in target domain just for training of `Oracle` experiments, which are fully supervised training and testing of the target domain.

**Comment by DuyunliangToon** [comment]
Thanks for your guidance.
